src/main/java/view/InputView.java

```diff
@@ -0,0 +1,78 @@
+package view;
+
+import domain.Crew;
+import domain.Day;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+public class InputView {
+    public static final String MENU_REGEX = "[1234Qq]";
+    private final Scanner scanner = new Scanner(System.in);
+
+    public String readMenu(LocalDate date) {
+        System.out.printf("오늘은 %d월 %d일 %s입니다. 기능을 선택해 주세요.%n" +
+                "1. 출석 확인%n" +
+                "2. 출석 수정%n" +
+                "3. 크루별 출석 기록 확인%n" +
+                "4. 제적 위험자 확인%n" +
+                "Q. 종료%n", date.getMonthValue(), date.getDayOfMonth(), Day.getDay(date).getName());
+        String input = scanner.nextLine();
+        validateMenuInput(input);
+        return input;
+    }
+
+    public Crew readNickname() {
+        System.out.println("\n닉네임을 입력해 주세요.");
```

Comment:
`System.lineSeparator()` 를 사용하면 OS 에 구애받지 않고 줄바꿈이 가능합니다.

```suggestion
        System.out.println(System.lineSeparator() + "닉네임을 입력해 주세요.");
```